sdk/src/metrics/export/periodic_exporting_metric_reader.cc

```diff
@@ -151,8 +149,11 @@ void PeriodicExportingMetricReader::DoBackgroundWork()
       worker_thread_instrumentation_->AfterWait();
     }
 #endif /* ENABLE_THREAD_INSTRUMENTATION_PREVIEW */
-
-  } while (IsShutdown() != true);
+    if(IsShutdown())
+    {
+      return;
```

Comment:
Don't we need call `worker_thread_instrumentation_->OnEnd()` below?

Reply:
+1, probably we can just `break` from here instead of `return`.